packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -82,16 +89,23 @@ function renderFooter(cta, type, href, Icon) {
   return (
     <footer className={`${prefix}--card__footer`}>
       {type !== 'link' ? (
-        <CTA style="text" type={cta.type} href={href} copy={cta.copy} />
+        <CTA
+          style="text"
+          type={cta.type}
+          href={href}
+          copy={cta.copy}
+          customClassname={`${prefix}--card__cta`}
```

Comment:
We need to align on the naming of this prop. Some places we're calling it `customClassname`, and other places we're calling it `className`.

Reply:
@kennylam @andysherman2121 @RaphaelAmadeu @mkothur what do you guys think? when passing a classname to a child component, should we name the prop `customClassname` or `className`?

Reply:
> @kennylam @andysherman2121 @RaphaelAmadeu @mkothur what do you guys think? when passing a classname to a child component, should we name the prop `customClassname` or `className`?

For me `customClassname` is make sense to me to differentiate with actual `className` for child component, I gave for `ContentGroup`.

Reply:
yeah I agree with that as well